pkg/bundle/read.go

```diff
@@ -247,6 +250,39 @@ func read(ctx context.Context, name, baseDir string, bundleSpecReader io.Reader,
 	return New(def, scans...)
 }
 
+// fixAndApplyHelmSpec checks if the spec's Helm repo, if there is one, ends with a '/' and appends one if not. See chartURL
+// This function also propagates root Helm chart properties to the child targets and performs the repo check.
+// Note: The repo and version properties of child targets are only propagated if either property is overridden. This
+// allows child targets to specify a path to a chart, to specify a url to the chart(e.g. oci://), etc.
+func fixAndApplyHelmSpec(spec *fleet.BundleSpec) {
+	if spec.Helm != nil && spec.Helm.Chart != "" {
+		if spec.Helm.Repo != "" && !strings.HasSuffix(spec.Helm.Repo, "/") {
+			spec.Helm.Repo = spec.Helm.Repo + "/"
```

Comment:
`spec.Helm.Repo` should already have a `/` added by `chartURL`, so I suppose only `target.Helm.Repo` would need that logic, but maybe we should fix the requirement for a `/`  instead if it is really mandatory for the rest of the fleet code.

Reply:
I tried to use this code and accompioning `if` at the end of the for loop to standardize the repo option with '/' suffix throughout Fleet, so that the logic for verifying the repo ended in '/' for `spec.Helm.Repo` and `target.Helm.Repo` was all in one place. Although that can definitely be made more clear. If that stays, it should probably be extracted to a separate code block or function.

Would the user be required in the `fleet.yaml` to always end the repo in a '/'?

Reply:
> Would the user be required in the `fleet.yaml` to always end the repo in a '/'?

No, I do not think so. I do not know why this is added all the time by chartURL, maybe for concatenating, but in this case it should be handled there and not just be added by default.
Unless Helm would require it, which is not the case to my knowledge.